modules/ROOT/pages/idempotent-message-validator.adoc

```diff
@@ -4,41 +4,69 @@ include::_attributes.adoc[]
 endif::[]
```

Comment:
Instructions for edit:

Link to the PR: https://github.com/mulesoft/docs-mule-runtime/pull/1160
Type of edit requested : Standard
Due date: Friday 01/31 (preferred: Tuesday 01/28)